test/e2e-common.sh

```diff
@@ -39,7 +39,26 @@ function knative_setup() {
   echo ">> Starting Knative Eventing"
   echo "Installing Knative Eventing"
   ko apply -f ${EVENTING_CONFIG} || return 1
+
+  # Force tracing to 100%.
+  kubectl apply -f - << END
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: knative-eventing
+  name: config-tracing
+data:
+  backend: stackdriver
+  zipkin-endpoint: "http://zipkin.istio-system.svc.cluster.local:9411/api/v2/spans"
+  debug: "true"
+  sample-rate: "1.0"
+END
   wait_until_pods_running knative-eventing || fail_test "Knative Eventing did not come up"
+
+  echo "Installing Knative Monitoring"
+  kubectl create namespace istio-system
```

Comment:
I noticed only `zipkin` is installed in `istio-system` namespace, `prometheus`, `elasticsearch-logging`, etc. are installed in `knative-monitoring`.
Does it have to, or is it a bug? @yanweiguo can you please confirm?

Reply:
The [comment](https://github.com/knative/serving/blob/master/config/monitoring/tracing/zipkin/100-zipkin.yaml#L19) explains why we install `zipkin` in `istio-system`. We can revisit this whether it is doable to move to `knative-monitoring` with current `istio` version.